app/assets/javascripts/controllers/reconfigure/reconfigure_form_controller.js

```diff
@@ -211,6 +291,8 @@ ManageIQ.angular.app.controller('reconfigureFormController', ['$http', '$scope',
       add_remove: 'add'});
     vm.resetAddValues();
     vm.updateDisksAddRemove();
+
+    vm.cb_disks = vm.reconfigureModel.vmAddDisks.length > 0  || vm.reconfigureModel.vmRemoveDisks.length > 0;
```

Comment:
I inadvertantly messed with cb_disks. I will remove line 295 and 334 from my PR.